### PR TITLE
FEATURE: Custom Class for both slider and slides

### PR DESCRIPTION
**Pull request by abteilung**
Custom Classes for both slider and individual slides. Now you can apply custom styles to individual slide(r)s.

### CODE REVIEW

File: Configuration/NodeTypes.Content.Slide.yaml [hunk 1]
```diff
@@ -13,3 +14,7 @@
           label: 'i18n'
           icon: 'icon-picture'
           tab: 'default'
+        'slick-appearance':
+          label: 'i18n'
+          icon: 'icon-images'
```
Comment (breadlesscode) [inline Configuration/NodeTypes.Content.Slide.yaml]: This icon should be icon-picture ;)

File: Resources/Private/Fusion/Slide.fusion [hunk 2]
```diff
@@ -7,8 +7,11 @@ prototype(Noerdisch.Slick:Content.Slide) < prototype(Neos.Neos:ContentComponent)
     renderer = Neos.Fusion:Tag {
         content = Neos.Neos:ContentCollection {
             attributes {
-                class = 'slide__inner'
-                class.@process.collectionClass >
+                class = Neos.Fusion:RawArray {
+                    class = 'slide__inner'
+                    additionalClass = ${node.properties.additionalClass}
```
Comment (breadlesscode) [inline Resources/Private/Fusion/Slide.fusion]: Please use flow query to access the property.

File: Resources/Private/Fusion/Slider.fusion [hunk 3]
```diff
@@ -7,6 +7,7 @@ prototype(Noerdisch.Slick:Content.Slider) < prototype(Neos.Neos:ContentComponent
         @context.node = ${ node }
     }
     renderer =  Neos.Neos:ContentCollection {
+        attributes.class = ${node.properties.additionalClass}
```
Comment (breadlesscode) [inline Resources/Private/Fusion/Slider.fusion]: Same here. Please use flow query to access the property

**Review by breadlesscode — CHANGES_REQUESTED** [review]
Please fix that small issues :) Thanks for your PR! 👍

**Review by breadlesscode — APPROVED** [review]

**Review by breadlesscode — APPROVED** [review]

**Review by breadlesscode — APPROVED** [review]

**Comment by markusguenther** [comment]
Created a small issue to improve the docs. But great that you have added this feature :)
Thank you 🙏

**Comment by abteilung** [comment]
@markusguenther There's a typo in the flow query string in Slider.fusion. Sorry about that.

**Comment by breadlesscode** [comment]
@abteilung can you send a PR? Please use `yarn commit` for any commit :+1:

**Pull request merged**